sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -232,6 +256,20 @@ static const struct snd_soc_acpi_link_adr adl_rvp[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr adl_chromebook_base[] = {
+        {
+                .mask = BIT(0),
+                .num_adr = ARRAY_SIZE(rt5682_0_adr),
+                .adr_d = rt5682_0_adr,
+        },
+        {
+                .mask = BIT(2),
```

Comment:
are you sure? This would mean using link2, which IIRC is mutually exclusive with DMICs

Reply:
I meant that there are mutual exclusions between some SoundWire link and DMIC0/1 interfaces. You want to double check that the mapping allows for 2 SoundWire links and 2 DMIC links (total of 8 wires for audio, 2 SoundWire clk, 2 SoundWire data, 2 DMIC data (DMIC clocks could be combined or independent depending on use cases)

Reply:
Yes, they are mutually exclusive. Yes mapping allows us to use SDW0 and SDW2 links and 2 DMICS with independent clocks and Data.